src/coord/src/coord.rs

```diff
@@ -5498,7 +5498,6 @@ mod timeline {
                     }
                     assert!(ts.less_than(&next));
                     self.state = TimestampOracleState::Writing(next.clone());
-                    self.advance_to = Some(next.clone());
```

Comment:
This is instead the line that would make reads after writes have potentially high latency.